dist/index.html

```diff
@@ -9,14 +9,22 @@
 <script defer src="main.js"></script></head>
 <body>
   <header>
-    
+    <h1> GET MOVIES</h1>
+    <nav>
+      <a href="index.html"> Home </a>
+      <a href="index.html"> Latest Movies </a>
+    </nav>
   </header>
   <main id ="main">
     <div class="movies"> 
       <div class="movie-info">
       </div>
     </div>
   </main>
+  <section class = "comment-popup">
+
+
+  </section>
```

Comment:
- Your HTML code looks perfects. To make it more clear , kindly remove this empty space.